apophis/cmds/music.py

```diff
@@ -194,6 +195,13 @@ def get_next_song() -> Optional[int]:
         return song
 
 
+async def random_song() -> int:
```

Comment:
This doesn't need to be `async` since its not performing any asynchronous computations